js/main.js

```diff
@@ -1,19 +1,119 @@
-// Функция, возвращающая случайное число
-const getRandomInteger = function (min, max) {
-  if (min >= 0 && max >= 0) {
-    return Math.floor(Math.random() * (max - min + 1) + min);
+const AUTHORS = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+const TYPES = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+
+const CHECKIN = ['12:00', '13:00', '14:00'];
+
+const CHECKOUT = ['12:00', '13:00', '14:00'];
+
+const FEATURES = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+
+const PHOTOS = ['https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg'];
+
+const SIMILAR_AD_COUNT = 10;
+
+const ADDRESS_LOCATION_X = 60;
+
+const ADDRESS_LOCATION_Y = 590;
+
+const Prices = {
+  PRICE_MIN: 500,
+  PRICE_MAX: 5000,
+};
```

Comment:
```suggestion
const Prices = {
  MIN: 500,
  MAX: 5000,
};
```

Иначе масло масляное получается.
https://up.htmlacademy.ru/javascript/23/criteries#d3 вот тут объясняют

Reply:
поправить все остальные аналогично